src/Symfony/Bridge/Monolog/Handler/NotifierHandler.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Monolog\Handler;
+
+use Monolog\Handler\AbstractHandler;
+use Monolog\Logger;
+use Symfony\Component\Notifier\Notification\Notification;
+use Symfony\Component\Notifier\Notifier;
+use Symfony\Component\Notifier\NotifierInterface;
+
+/**
+ * Uses Notifier as a log handler.
+ *
+ * @author Fabien Potencier <fabien@symfony.com>
+ */
+class NotifierHandler extends AbstractHandler
+{
+    private $notifier;
+
+    public function __construct(NotifierInterface $notifier, int $level = Logger::ERROR, bool $bubble = true)
+    {
+        $this->notifier = $notifier;
+
+        parent::__construct($level < Logger::ERROR ? Logger::ERROR : $level, $bubble);
+    }
+
+    public function handle(array $record): bool
+    {
+        if (!$this->isHandling($record)) {
+            return false;
+        }
+
+        $this->notify([$record]);
+
+        return !$this->bubble;
+    }
+
+    public function handleBatch(array $records): void
+    {
+        if ($records = array_filter($records, [$this, 'isHandling'])) {
+            $this->notify($records);
+        }
+    }
+
+    private function notify(array $records): void
+    {
+        $record = max(array_column($records, 'level'));
+        if (($record['context']['exception'] ?? null) instanceof \Throwable) {
+            $notification = Notification::fromThrowable($record['context']['exception']);
+        } else {
+            $notification = new Notification($record['message']);
+        }
+
+        $notification->importanceFromLogLevelName(Logger::getLevelName($record['level']));
+
+        try {
+            $this->notifier->send($notification, ...Notifier::getAdminReceivers());
+        } catch (\Throwable $e) {
```

Comment:
I'm always suspicious when a code catches `Throwable`. Shouldn't it be `Exception` instead, and let developer mistakes bubble to their face?

Reply:
It's in an Monolog Handler, so something bad is already happened, and maybe in production. We should not trigger an infinite loop. So `\Throwable` is fine to me

Reply:
I'm really doubtful: that could also lead to a missed silent error during dev.

Reply:
Monolog has an handler dedicated at swallowing errors in handlers.

Reply:
Removed all the try/catch, Monolog does handle exception nicely already.